runbot_cla/runbot.py

```diff
@@ -0,0 +1,44 @@
+# -*- encoding: utf-8 -*-
+
+import glob
+import logging
+import os
+import re
+
+import openerp
+
+_logger = logging.getLogger(__name__)
+
+class runbot_build(openerp.models.Model):
+    _inherit = "runbot.build"
+
+    def job_15_check_cla(self, cr, uid, build, lock_path, log_path):
+        # TODO manage exception in schedule and remove excepts
+        build._log('check_cla', 'Check cla')
+        if build.repo_id.token:
+            cla_glob = build.path("doc/contributors/*.md")
+            cla = ''.join(open(f).read() for f in glob.glob(cla_glob))
+            cla = cla.lower()
+            mo = re.search('[^ <@]+@[^ @>]+', build.author)
```

Comment:
This regex is too fuzzy.
Use `tools.email_split`
(see https://github.com/odoo/odoo/commit/005e24fada851efaed112fce4244a3af225d8f74)